scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -214,7 +214,7 @@ class FossId internal constructor(
         endTime: Instant = Instant.now(),
         issue: Issue
     ) = ScanSummary(
-        startTime, endTime, "", sortedSetOf(), sortedSetOf(), listOf(issue)
+        startTime, endTime, "", sortedSetOf(), sortedSetOf(), issues = listOf(issue)
```

Comment:
Using named parameter is required because the signature of  `ScanSummary` has changed.
Otherwise the compiler throws:
`Type mismatch: inferred type is List<Issue> but SortedSet<SnippetFinding> was expected`